otelcollector/configmapparser/prometheus-config-merger.rb

```diff
@@ -117,7 +117,7 @@ def AppendMetricRelabelConfig(yamlConfigFile, keepListRegex)
           if metricRelabelCfgs.nil?
             scfg["metric_relabel_configs"] = keepListMetricRelabelConfig
           else
-            scfg["metric_relabel_configs"] = metricRelabelCfgs.concat(keepListMetricRelabelConfig)
+            scfg["metric_relabel_configs"] = keepListMetricRelabelConfig.concat(metricRelabelCfgs)
```

Comment:
this file isn't being used in the new CCP image. I have another one called -> prometheus-ccp-config-merger.rb but maybe this change isn't required anymore? otherwise this PR looks good to me!